act-rules-format.bs

```diff
@@ -140,43 +140,34 @@ Component properties starting with `aria-` MUST exist in the list of WAI-ARIA at
 
-This can be used together, such as in the following example which uses CSS selector syntax to locate elements, combined with a plain language description to further filter the nodes.
+Applicability SHOULD be described unambiguously in plain language. When a formal syntax (such as a CSS selector or XPATH) can be used, that (part of the) applicability MAY use that syntax in addition to the plain language description. While testing, if nothing on in the test subject matches the applicability of the rule, the result is `inapplicable`.
 
 <div class=example>
-A rule for HTML `img` elements has the following selector:
+A rule for HTML `img` elements without alt has the following applicability:
 
-1. Select elements matching the following CSS Selector: `img:not([alt=""])`
-2. Exclude any elements that have `visibility:hidden` or `display: none`
+Each `img` element without an `alt` attribute (e.g. `img:not([alt])` in CSS).
 
+**Exception**: Any element removed from the accessibility tree or that had it's role changed.
 </div>
 
 
-Test Execution Steps {#test-proc-steps}
------------------------------
+Expectations
+------------
 
-Each rule contains one or more test execution steps written in plain language. Each test execution step MUST provide the following:
+Each rule MUST contain one or more expectations. An expectation is a statement that must be true about each test target (see Applicability). Each expectation MUST be written in plain language.
 
-* Test mode identifying whether test execution step is automated, semi-automated or requires manual testing.
-* Test execution which clearly outlines the actions necessary to reach the result.
-* Test result indicating whether test execution step passed or failed.
-  * Description of the error causing any failing result.
-
-Test execution step results are combined with other test execution step results to provide a cumulative outcome, pass or fail, of the rule. When a rule has multiple test execution steps, the results of all test execution steps are combined to give a single result for each test target. The results are cumulative, meaning that as long as one test execution step passes, the rule has passed.
+When all expectations are true for a test target it `passed` the rule. If one or more expectations is not true, the test target `failed` the rule. If the rule is part of a rule group, a test target that `failed` a rule may still meet the requirement (see rule grouping for details).
 
 <div class=example>
-A rule for HTML `img` elements has two test execution steps:
-
-1. Check if there is a text alternative
-2. Check if the image is marked as decorative.
+A rule for HTML `img` elements without alt has the following two expectations:
 
-If either one of these passed, the rule has passed. Only if both fail, does that element fail the rule.
+1. The image has an ARIA label, giving it an accessible name
+2. The accessible name is descriptive of the image
 </div>
 
```

Comment:
These two examples make it very difficult to compare, as they are not equivalent.